project/src/index.tsx

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './components/app/app';
 
+const Settings = {
+  OffersCount: 5,
+} as const;
+
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <App
+      offersCount={Settings.OffersCount}
+    />
```

Comment:
Когда пропсов мало лучше писать в строку, а когда 3-4 и больше то уже в столбик
```suggestion
    <App offersCount={Settings.OffersCount} />
```